ext/web/02_timers.js

```diff
@@ -35,7 +35,7 @@ function checkThis(thisArg) {
 function setImmediate(callback, ...args) {
   if (args.length > 0) {
     const unboundCallback = callback;
-    callback = () => ReflectApply(unboundCallback, window, args);
+    callback = () => ReflectApply(unboundCallback, globalThis, args);
```

Comment:
i'm not sure why these had to be changed, i'm pretty sure this file should be deno mode.